src/main/java/teammates/ui/webapi/CreateFeedbackSessionLogAction.java

```diff
@@ -24,15 +24,16 @@ void checkSpecificAccessControl() {
 
     @Override
     public JsonResult execute() {
+
         String fslType = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_LOG_TYPE);
         FeedbackSessionLogType convertedFslType = FeedbackSessionLogType.valueOfLabel(fslType);
         if (convertedFslType == null) {
             throw new InvalidHttpParameterException("Invalid log type");
         }
 
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
-        String fsName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
         String studentEmail = getNonNullRequestParamValue(Const.ParamsNames.STUDENT_EMAIL);
+        String fsName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
```

Comment:
All unrelated changes?

Reply:
We need `fsName` for `FeedbackSessionLogEntryAttributes` in `createFeedbackSessionLog` of `LocalLoggingService`